nemo/collections/nlp/models/token_classification/ner_model.py

```diff
@@ -54,36 +54,40 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             data_dir=cfg.data_dir, modes=["train", "test", "dev"], pad_label=cfg.pad_label
         )
         self.data_dir = cfg.data_dir
-        self.model_cfg = cfg
+        # self.model_cfg = cfg
 
         self.tokenizer = get_tokenizer(
             tokenizer_name=cfg.language_model.tokenizer,
             pretrained_model_name=cfg.language_model.pretrained_model_name,
-            vocab_file=cfg.language_model.vocab_file,
-            tokenizer_model=cfg.language_model.tokenizer_model,
+            vocab_file=self.register_artifact(
+                conf_path='language_model.vocab_file', src=cfg.language_model.vocab_file
```

Comment:
Lets use the full `config_path=` rather than `conf_path`.

Reply:
changed